### ISSUE TITLE
Enhancement: Screen options other than the current screen

### ISSUE THREAD

**Issue by evilguy422** [prusa3d/Original-Prusa-SL1#10]
I think it would be great for Prusa to give upgrades we can buy as replacement screens for the SL1 such as either a mono screen or a 4k screen. Would be very doable and Im guessing software side should be much of an issue either. It would be great for future proofing the machine since its not a cheap one. I would much rather spend a bit to upgrade the screen than to get another machine. Its not like replacing an anycubic with a sonic mini, cost wise. Id love to keep my prusa for forever if I can because I love it, so I hope they consider doing this if its not already planned. 

Thank you

**Comment by Lichtjaeger** [comment]
I expect/hope they will handle it like they do with their i3 machines. A complete SL2 upgrade kit to improve the current model and add edge technologies.

If they would offer a lot of individual component upgrades, they would have to offer firmware versions and profiles for all possible combinations. So a complete pack would make more sense.

But because the SL1 isn't as popular as the i3 and the Mini are and the machine is only a year old, I assume the SLX hardware development will not have a high priority at the moment.

**Comment by evilguy422** [comment]
Yeah you points make a lot of sense. The SL1 is by far the best build quality printer out there so Id hate to see it become outdated. The price is a bit steep but when you see everything that went into it it makes sense. An upgrade kit would be cool to see, but other than the screen I dont personally see what else they could upgrade.

**Comment by Lichtjaeger** [comment]
Flexplate system would be nice.
And I have still a problem with resin accumulating on the top of the print platform. At the moment I use [this](https://www.prusaprinters.org/prints/5833-prusa-sl1-platform-drip-adapter) to let it drip off.
Maybe the next platform could be tapered like the Elegoo platforms or a build-in mechanic to tilt the platform.

**Comment by Lichtjaeger** [comment]
And there I have directly even more suggestions:
- A replacement screen could be much cheaper if it didn't have to come with a milled aluminium frame
- I'm currently working with the Druckwege High-Temperature resin. It prints much better when it is warm. The CW1's preheat function helps at first, but the resin cools down as it prints. So a heated tank would be helpful.

These improvements would give an SL2 an edge over those low-cost printers.
